loader/src/main/scala/com/snowplowanalytics/snowplow/storage/bigquery/loader/Loader.scala

```diff
@@ -71,8 +70,12 @@ object Loader {
       .flatMap {
         case (Right(row), ctx) =>
           ctx.output(ObservedTypesOutput, row.inventory)
-          val latency = (Instant.now().getMillis - row.collectorTstamp.getMillis) / 1000
-          latencyDistribution.update(latency)
+          val diff = Instant.now().getMillis - row.collectorTstamp.getMillis
+          latency.update(diff) match {
```

Comment:
Where this `latency` came from?

Reply:
It's a `MetricRegistryOps` instance imported with the `metrics` object.

Reply:
Ah ok, sorry, I misinterpreted what `MetricRegistryOps` is.

Reply:
That puts your other comment about making it implicit in context.

Reply:
Yes.